core/data/tests/test_algebraic_enum_case_name_support/output.py

```diff
@@ -0,0 +1,55 @@
+"""
+ Generated by typeshare 1.12.0
+"""
+from __future__ import annotations
+
+from enum import Enum
+from pydantic import BaseModel, ConfigDict
+from typing import List, Union
+
+
+class ItemDetailsFieldValue(BaseModel):
+    pass
+
+class AdvancedColorsTypes(str, Enum):
+    STRING = "string"
+    NUMBER = "number"
+    NUMBER_ARRAY = "number-array"
+    REALLY_COOL_TYPE = "reallyCoolType"
+
+class AdvancedColors(BaseModel):
+    model_config = ConfigDict(use_enum_values=True)
```

Comment:
Any particular reason to use `use_enum_values`? Can the pydantic->JSON layer handle that conversion for us?

Reply:
Understood, thanks for the explaination.